services/response_code.proto

```diff
@@ -1179,4 +1179,26 @@ enum ResponseCodeEnum {
    * No allowances have been specified in the approval/adjust transaction.
    */
   EMPTY_ALLOWANCES = 295;
+
+  /**
+   * Spender is repeated more than once in Crypto or Token or NFT allowance lists in a single
+   * CryptoApproveAllowance or CryptoAdjustAllowance transaction.
+   */
+  SPENDER_ACCOUNT_REPEATED_IN_ALLOWANCES = 296;
+
+  /**
+   * Serial numbers are repeated in nft allowance for a single spender account
+   */
+  REPEATED_SERIAL_NUMS_IN_NFT_ALLOWANCES = 297;
+
+  /**
+   * Fungible common token used in NFT allowances
+   */
+  FUNGIBLE_COMMON_TOKEN_IN_NFT_ALLOWANCES = 298;
```

Comment:
nit : could be named without out `COMMON`

Reply:
Done.